docs/fundamentals/flow/topology.md

```diff
@@ -218,7 +218,11 @@ This is helpful in two situations:
 
 Then splitting the load across two or more machines yields better results.
 
-For Shards, you can define which shard (instance) will receive the request from its predecessor. This behaviour is called `polling`. By default `polling` is set to `ANY`, which means only one shard will receive a request. If `polling` is to `ALL` it means that all Shards will receive a request.
+For Shards, you can define which shard (instance) will receive the request from its predecessor. This behaviour is called `polling`. `ANY` means only one shard will receive a request and `ALL` means that all Shards will receive a request. Polling can be configured per endpoint (like `/index`) and Executor.
+By default the following `polling` is applied:
+- `ANY` for endpoints at `/index`
+- `ALL` for endpoints at `/search`
+- `ANY` for all other endpoints
```

Comment:
maybe we should mention that `/index` will always be 'any' and `/search` will always be `all` regardless of the `polling` value, unless you modify them in `endpoint_polling` ?

Reply:
i tried to say that with this paragraph, but I will make it a bit more clear

Reply:
I've made it more explicit now in the docs. I think its still confusing, but it will be hard to explain in the docs I think.
I think we should probably merge the two options into a single one, I've created a follow up issue for this https://github.com/jina-ai/internal-tasks/issues/359